### DISCUSSION TITLE
What is the best way for a beginner to start contributing to the cFS project on GitHub?

### DISCUSSION THREAD

**Discussion by madhu22222** [nasa/cFS#806] [Q&A] [answered]
As a beginner interested in contributing to the NASA cFS project, I’m unsure where to start. Is there a specific process or guideline I should follow?

**Comment by madhavi-peddireddy** [comment, marked as answer]
To start contributing to the cFS project, begin by reviewing the project's documentation and contribution guidelines available in the repository. Familiarize yourself with the codebase and look for beginner-friendly issues labeled with tags like “good first issue” or “help wanted.” Engaging with the community through discussions and asking questions will also help. Additionally, try running the code locally and experimenting with small changes to build your confidence before submitting pull requests.

**Comment by irowebbn** [comment]
Hi, the instructions for contributing are [here](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#ways-to-contribute). The most important points:
- All contributors need to sign a contributor license agreement (CLA)
- Pull requests are reviewed weekly at a [configuration control board](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process) meeting. The regular maintainers will look at the PRs and decide which ones to prioritize reviewing. 

The backlog of issues has lots of stuff that may be obsolete now, so you may want to ask a regular contributor what would be most helpful. Keep in mind that some parts of cFS are not beginner-friendly for editing. It would help a lot if you've worked with cFS on a real system.

(Yes, I know the original question was a sockpuppet of the answerer, but I wanted to put some non-AI generated information up with relevant info in case anyone was searching).
